tests/test_pyqvm.py

```diff
@@ -365,7 +365,7 @@ def circuit():
     @pytest.mark.parametrize("device", ["2q-pyqvm"])
     def test_one_qubit_wavefunction_circuit(self, device, shots):
         """Test that the wavefunction plugin provides correct result for simple circuit."""
-        shots = 100000
+        shots = 10000
```

Comment:
Has this just been decreased to reduce the test suite time?

Reply:
Yes. In the previous run, for some reason, the testing was killed at one of the PyQvm tests